utils/types.go

```diff
@@ -94,6 +94,11 @@ type KongClientConfig struct {
 	HTTPClient *http.Client
 
 	Timeout int
+
+	SessionFilePath string
```

Comment:
```suggestion
	CookieJarPath string
```
Cookies can do more than sessions.

Reply:
Done